scripts/LargeScaleStructures/data_stitching.py

```diff
@@ -539,14 +540,14 @@ def get_scaled_data(self, workspace=None):
                 e.extend(_e)
                 dx.extend(_dx)
 
-        zipped = zip(x, y, e, dx)
+        zipped = list(zip(x, y, e, dx))
 
         def cmp(p1, p2):
             if p2[0] == p1[0]:
                 return 0
             return -1 if p2[0] > p1[0] else 1
-        combined = sorted(zipped, cmp)
-        x, y, e, dx = zip(*combined)
+        combined = sorted(zipped, key=cmp_to_key(cmp))
```

Comment:
why not `combined = sorted(zipped, key=(lambda x:x[0]))` and the you don't need to transform zipped into list on line 543, and don't need to write a `cmp` function, and don't need to use `cmp_to_key`